bikeshed/boilerplate/whatwg/header-RD.include

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta name="theme-color" content="#3c790a">
+<title>[TITLE]</title>
+<link href="https://resources.whatwg.org/spec.css" rel="stylesheet">
+<link href="https://resources.whatwg.org/review-draft.css" rel="stylesheet">
+<link href="[LOGO]" rel="icon">
+
+<body class="h-entry status-[STATUS]">
+<div class="head">
+  <p data-fill-with="logo"></p>
+  <hgroup>
+    <h1 id="title" class="p-name no-ref">[SPECTITLE]</h1>
+    <h2 id="subtitle" class="no-num no-toc no-ref">[LONGSTATUS] — Published <time class="dt-updated" datetime="[ISODATE]">[DATE]</time></h2>
+  </hgroup>
+  <details class="annoying-warning" open>
+    <summary>This Is a Review Draft of the Standard</summary>
```

Comment:
This is also wrong for the other warnings. We shouldn't use titlecase here. This should be "This is a Review Draft of the standard".